src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -69,6 +78,7 @@ export class CommentEditorComponent implements OnInit {
 
     this.initialSubmitButtonText = this.submitButtonText;
     this.commentField.setValidators([Validators.maxLength(this.maxLength)]);
+    this.history = new UndoRedo<textEntry>(50, {text: "", selectEnd: 0, selectStart: 0});
```

Comment:
I swapped to passing the initial value as the initial value of the text area instead of an empty string. This should resolve the issue when there are already preloaded text in the text-area